spec/user_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'

```

Comment:
- [ ] I ran your tests locally in my machine and they all failed as seen in the screenshot below:
![image](https://github.com/rodrigopenayo98/Blog-App-Rod/assets/45460192/0a13cb5e-a865-4ba9-b9bb-80e822738770)